deploy/dev-ngsa-asb-eastus/ngsa/dev/ngsa/ngsa-cosmos.yaml

```diff
@@ -43,6 +43,8 @@ spec:
           - az-eastus-dev
           - --region
           - eastus
+          - --cosmos-auth-type
```

Comment:
This can be removed. Let AutoGitOps update the deployment yaml files.

Reply:
This has been fixed.